client/state/data-layer/wpcom/read/recommendations/sites/index.js

```diff
@@ -0,0 +1,67 @@
+
+/**
+ * External Dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	READER_RECOMMENDED_SITES_REQUEST
+} from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { receiveRecommendedSites } from 'state/reader/recommended-sites/actions';
+
+export function requestRecommendedSites( { dispatch }, action, next ) {
+	const { seed = 1, number = 10, offset = 0 } = action.payload;
+	dispatch( http( {
+		method: 'GET',
+		path: '/read/recommendations/sites',
+		query: { number, offset, seed, posts_per_site: 0 },
+		apiVersion: '1.2',
+		onSuccess: action,
+		onFailure: action,
+	} ) );
+	next( action );
+}
+
+export const fromApi = response => {
+	if ( ! response ) {
+		return [];
+	}
+
+	return map( response.sites, site => ( {
+		feedId: site.feed_id,
+		blogId: site.blog_id,
+		title: site.blog_title,
+		url: site.blog_url,
```

Comment:
because they're part of the response? I wasn't aware that we're no longer using them in the UI. If we keep going that road, we should limit what we ask for with `fields` to avoid building up the posts in the first place.